exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,7 @@
-// Declare your function here
+function createLongGreeting (firstName, age) {
+    let message = "Hello, my name is " + firstName + " and I'm " + age + " years old";
+    return message;
```

Comment:
I would avoid assigning the variable here and just return the statement directly.